deployment/import_instructions.md

```diff
@@ -0,0 +1,114 @@
+# Import Instructions
+
+## From `pete-sls` branch:
+
+1. Deploy sls to get it ready for deletion with retained resources configured for import
+
+```
+./run deploy --stage <YOUR_BRANCH_NAME>
+```
+
+2. Collect information about the resources we're going to be importing into the new cdk stack.
+```
+cloudfront.Distribution -
+cognito.UserPool -
+```
+
+3. Destroy sls
+
+```
+./run destroy --stage <YOUR_BRANCH_NAME>
+```
+
+:warning: Make sure that all sls associated stacks have been fully destroyed before proceeding.
+
+
+## From `jon-cdk` branch:
+
+
+1. Create just the new cdk stack without anything inside of it.
+
+```bash
+WITHOUT_IMPORTS=true ./run deploy --stage <YOUR_BRANCH_NAME>
+```
+
+2. Now import all the serverless ejected resources.
+
+```bash
+WITH_IMPORTS=true PROJECT=seds cdk import --context stage=<YOUR_BRANCH_NAME> --force
+```
+As this import occurs you'll have to provide the information you gathered just before destroying the serverless stacks. For the dynamo tables the default should be correct but read closely to be sure.
+
+3. Run a deploy on that same imported resource set.
+
+```bash
+WITH_IMPORTS=true ./run deploy --stage <YOUR_BRANCH_NAME>
+```
+
+4. Run a full deploy by kicking off the full cdk deploy via Github Action. Permissions for individual developers are limited so you must use Github Action to do this part.
```

Comment:
we can't do this from our local?

Reply:
Just confirmed, you're right we can. I've updated the instructions accordingly.